### DISCUSSION TITLE
Mark post as Read in Anonymous mode

### DISCUSSION THREAD

**Discussion by Humorist2601** [Docile-Alligator/Infinity-For-Reddit#983] [Ideas]
Hello, I know that marking a post as read is already implemented, but it should also work in Anonymous mode like in Slide for Reddit.
Thanks

**Comment by Fehertukor** [comment]
Hello, I would love this feature too in anonymous mode. Please if you have time implement it!

**Comment by Wladefant** [comment]
agree
